1.16/Dockerfile

```diff
@@ -0,0 +1,92 @@
+FROM centos/s2i-core-centos7:1
+
+# RHSCL rh-nginx114 image.
+#
+# Volumes:
+#  * /var/opt/rh/rh-nginx114/log/nginx/ - Storage for logs
+
+EXPOSE 8080
+EXPOSE 8443
+
+ENV NAME=nginx \
+    NGINX_VERSION=1.14 \
+    NGINX_SHORT_VER=114 \
+    VERSION=0
+
+ENV SUMMARY="Platform for running nginx $NGINX_VERSION or building nginx-based application" \
+    DESCRIPTION="Nginx is a web server and a reverse proxy server for HTTP, SMTP, POP3 and IMAP \
+protocols, with a strong focus on high concurrency, performance and low memory usage. The container \
+image provides a containerized packaging of the nginx $NGINX_VERSION daemon. The image can be used \
+as a base image for other applications based on nginx $NGINX_VERSION web server. \
+Nginx server image can be extended using source-to-image tool."
+
+LABEL summary="${SUMMARY}" \
+      description="${DESCRIPTION}" \
+      io.k8s.description="${DESCRIPTION}" \
+      io.k8s.display-name="Nginx ${NGINX_VERSION}" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.expose-services="8443:https" \
+      io.openshift.tags="builder,${NAME},rh-${NAME}${NGINX_SHORT_VER}" \
+      com.redhat.component="rh-${NAME}${NGINX_SHORT_VER}-container" \
+      name="centos/${NAME}-${NGINX_SHORT_VER}-centos7" \
+      version="${NGINX_VERSION}" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>" \
+      help="For more information visit https://github.com/sclorg/${NAME}-container" \
+      usage="s2i build <SOURCE-REPOSITORY> centos/${NAME}-${NGINX_SHORT_VER}-centos7:latest <APP-NAME>"
+
+ENV NGINX_CONFIGURATION_PATH=${APP_ROOT}/etc/nginx.d \
+    NGINX_CONF_PATH=/etc/opt/rh/rh-nginx${NGINX_SHORT_VER}/nginx/nginx.conf \
+    NGINX_DEFAULT_CONF_PATH=${APP_ROOT}/etc/nginx.default.d \
+    NGINX_CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/nginx \
+    NGINX_APP_ROOT=${APP_ROOT} \
+    NGINX_LOG_PATH=/var/opt/rh/rh-nginx${NGINX_SHORT_VER}/log/nginx
+
+RUN yum install -y yum-utils gettext hostname && \
+    yum install -y centos-release-scl-rh && \
```

Comment:
https://cbs.centos.org/repos/sclo7-rh-nginx116-rh-candidate/x86_64/os/ can be used meanwhile, till the packages are available on the mirror.

Reply:
Will do this in a follow-up PR, where I enable the rest of the image versions (fedora, centos, rhel7)

Reply:
This could be documented somewhere in the GitLab repository, I did not know about it.
Like all CentOS repositories which are used by SCL containers.

Reply:
I tend to check the postgresql distgen Dockerfile template where we have it hardcoded:
https://github.com/sclorg/postgresql-container/blob/master/src/Dockerfile#L61